mlflow/utils/rest_utils.py

```diff
@@ -361,10 +361,12 @@ def call_endpoint(host_creds, endpoint, method, json_body, response_proto, extra
     if extra_headers is not None:
         call_kwargs["extra_headers"] = extra_headers
     if method == "GET":
-        call_kwargs["params"] = json_body
+        if json_body:
+            call_kwargs["params"] = json_body
         response = http_request(**call_kwargs)
     else:
-        call_kwargs["json"] = json_body
+        if json_body:
```

Comment:
```suggestion
        if json_body is not None:
```

Reply:
Do we need to set params if json_body is {}?

Reply:
Doesn't hurt, right? I'm worried that `if json_body` might return `False` for some case that we actually care about. `if None` is more explicit

Reply:
Ok, will update this.

Reply:
Reverted this change. Setting `params` to `None` doesn't hurt.